tests/test_openapi.py

```diff
@@ -13,13 +13,14 @@
 
 @pytest.fixture()
 def marshmallow_plugin():
-    return MarshmallowPlugin()
+    return MarshmallowPlugin(schema_name_resolver=lambda x: None)
```

Comment:
Is the `schema_name_resolver` argument necessary?

Reply:
It's not 100% necessary, however as of apispec v1.0.0rc1, the response schema will be placed in the definitions section of the spec and then referenced from the appropriate place - meaning some of the ```test_responses``` tests  ```tests/test_openapi.py``` will fail, e.g.:

```
def test_responses(self, schemas, path, openapi):
        response = path['get']['responses']['default']
        assert response['description'] == 'a band'
        expected = openapi.schema2jsonschema(schemas.BandSchema)
>       assert response['schema'] == expected
E       AssertionError: assert {'$ref': '#/definitions/Band'} == {'properties':...pe': 'object'}
E         Left contains more items:
E         {u'$ref': u'#/definitions/Band'}
E         Right contains more items:
E         {u'properties': {'genre': {u'type': u'string'}, 'name': {u'type': u'string'}},
E          u'type': u'object'}
E         Use -v to get the full diff

tests/test_openapi.py:83: AssertionError
```

It was simply easier to set ```schema_name_resolver``` to a ```None```-returning callable, than to change the tests, although I'm happy to do that if you think that would be more appropriate :).